File: src/main/java/com/mymedia/nagasu/dto/ImageDetailsResponse.java
package com.mymedia.nagasu.dto;

import java.util.List;

public record ImageDetailsResponse(
        String id,
        String name,
        String title,
        String artist,
        List<String> tags,
        String description,
        String thumbnailUrl,
        List<String> images
) {
}

File: src/main/kotlin/com/mymedia/nagasu/dto/ImageDetailsResponse.kt
package com.mymedia.nagasu.dto

data class ImageDetailsResponse(
    val id: String,
    val name: String,
    val title: String,
    val artist: String,
    val tags: List<String>,
    val description: String,
    val thumbnailUrl: String,
    val images: List<String>
)